### PR TITLE
Add reliability parameter to cves and pipl integration

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-6735)

## Description
Add reliability parameter to integrations implementing a reputation command

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ShahafBenYakir — APPROVED** [review]

File: Packs/Pipl/Integrations/Pipl/README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Search for required query
 | Account.Names | unknown | Full names | 
 | Account.Phones | unknown | Phone numbers | 
 | Account.Usernames | unknown | Online platforms usernames | 
+
```
[the remark this answers is not in the document]
Comment (DinaMeylakh) [reply Packs/Pipl/Integrations/Pipl/README.md:62]: The whole configuration section was missing, I added it now, should I ask for a doc review?

File: Packs/Pipl/Integrations/Pipl/README.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
+Pipl provides reputation for email addresses and identity solutions.
```
Comment (ShirleyDenkberg) [inline Packs/Pipl/Integrations/Pipl/README.md:1]:
```suggestion
Pipl provides a reputation for email addresses and identity solutions.
```

**Comment by ShirleyDenkberg** [comment]
@ShahafBenYakir @GuyAfik Doc review completed.

**Comment by DinaMeylakh** [comment]
Failing on old js integration with no tests:
`[ERROR] Packs/Pipl/Integrations/Pipl/Pipl.yml: [IN140] - The integration Pipl is currently in skipped. Please add working tests and unskip. Skip comment: No instance`

Force merge needed.

**Pull request merged**